src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -6763,6 +6773,7 @@ static void addBreath(ChordRest* cr, const Fraction& tick, SymId breath)
         // b->setTrack(trk + voice); TODO check next line
         b->setTrack(cr->track());
         b->setSymId(breath);
+        b->setPlacement(PlacementV::ABOVE);
```

Comment:
This here actually fixes a downright bug, all breaths and casesuras from MusicXML get imported with a placement below!

Reply:
Replaced with a better mechanism, taken from master